Components/Sources/Components/ShimmerView/ShimmerConfig.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  [NAME].swift
+//  StackOv ([NAME] module)
+//
+//  Created by Владислав Климов
+//  Copyright © 2021 Erik Basargin. All rights reserved.
+//
+
+import SwiftUI
+import Combine
+
+public class ShimmerConfig: ObservableObject {
+    
+    // MARK: - Properties
+    
+    var bgColor: Color
+    var fgColor: Color
+    var shimmerColor: Color
+    var shimmerAngle: Double
+    var shimmerDuration: Double
+
+    @Published internal var isActive: Bool = false
+
+    // MARK: - Private Properties
+    
+    private var timer: AnyCancellable?
+    
+    // MARK: - Init & Deinit
```

Comment:
Please use defined MARKs of the project in new files. 🙃
My bad, I supposed to add this in the codestyle guide.